### PR TITLE
Remove six dependency during installation

### PR DESCRIPTION
**Pull request by pr33thi**
Remove six dependency during installation
Include a line in the config file to make sure a wheel gets added on pypi

### CODE REVIEW

File: couchdbkit/version.py [hunk 1]
```diff
@@ -4,6 +4,5 @@
 # See the NOTICE for more information.
 
 from __future__ import absolute_import
-from six.moves import map
-version_info = (0, 9, 1)
-__version__ =  ".".join(map(str, version_info))
+version_info = (0, 9, 2)
+__version__ = ".".join([str(vi) for vi in version_info])
```
Comment (nickpell) [inline couchdbkit/version.py]: Recommend setting your dev environment to include a newline at the end of file by default

File: couchdbkit/version.py [hunk 2]
```diff
@@ -4,6 +4,5 @@
 # See the NOTICE for more information.
 
 from __future__ import absolute_import
-from six.moves import map
```
Comment (nickpell) [inline couchdbkit/version.py]: Curious, what does this fix?

File: couchdbkit/version.py (hunk 1 above)
Comment (pr33thi) [reply couchdbkit/version.py]: good idea, Ill do this

File: couchdbkit/version.py (hunk 2 above)
Comment (pr33thi) [reply couchdbkit/version.py]: It errors if six isn't installed before couchdbkit-jsonobject

**Pull request merged**